packages/babel-core/src/transformation/normalize-file.js

```diff
@@ -157,7 +160,7 @@ function parser(
 
 // eslint-disable-next-line max-len
 const INLINE_SOURCEMAP_REGEX = /^[@#]\s+sourceMappingURL=data:(?:application|text)\/json;(?:charset[:=]\S+?;)?base64,(?:.*)$/;
-const EXTERNAL_SOURCEMAP_REGEX = /^[@#][ \t]+sourceMappingURL=(?:[^\s'"`]+?)[ \t]*$/;
+const EXTERNAL_SOURCEMAP_REGEX = /^[@#][ \t]+sourceMappingURL=(?<url>[^\s'"`]+?)[ \t]*$/;
```

Comment:
Dogfooding `named-capturing-regex` here.

The file size is bloated by helpers (6678 B -> 11276 B) but I think it is okay since `named-capturing-regex` is supported on Node.js 10+, which means we can get rid of the helpers in v8.